frontend/apps/filemanager/filemanagermenu.lua

```diff
@@ -231,7 +231,15 @@ function FileManagerMenu:setUpdateItemTable()
 
     -- main menu tab
     self.menu_items.open_last_document = {
-        text = _("Open last document"),
+        text = (function()
+            local last_file = G_reader_settings:readSetting("lastfile")
+            if last_file then
+                last_file = last_file:gsub(".+/", "") --remove path, leave filename
```

Comment:
You should use the util function for that. (E.g., think of Windows.)

http://koreader.rocks/doc/modules/util.html#splitFileNameSuffix

Reply:
And that `text = (function() ... end)()` is a bit strange - it should be `text_func = function() ... end` like most others.

Reply:
@poire-z, replaced with `text_func`.

@Frenzie  http://koreader.rocks/doc/modules/util.html#splitFileNameSuffix returns here:
`/media/marcin/Dane/koreader/test/juliet` (so path + filename -extension) 
should I create in util function such as getFileNameWithExtension that will return only filename without path?

Reply:
It's actually already there:
https://github.com/koreader/koreader/blob/8089d916a52de696b78f655b6e932753dd2ec9ff/frontend/util.lua#L402-L411

Reply:
@poire-z thanks